src/Application/View/projects/settings/states/_state.html.php

```diff
@@ -0,0 +1,30 @@
+<?php if ($first): ?>
+	<tr>
+		<td><?= h(mb_ucfirst($state['ticket_type'])); ?></td>
+<?php else: ?>
+	<tr>
+		<td class="empty"></td>
+<?php endif; ?>
+	<td><?=
+		$f->checkbox(
+			'States[' . $index . '][ticket_state]',
+			$state['ticket_state'],
+			$state['project_enabled'],
+			['value' => $state['ticket_state']] +
+				(($state['project_enabled'])?
+					['data-association-destroy' => 'States[' . $index . '][_destroy]'] :
+					[]),
+			false
+		) .
+		$f->hidden('States[' . $index . '][ticket_type]', $state['ticket_type']);
+	?></td>
+	<td class="right"><?php if ($state['service_executable']) {
+		echo $f->checkbox('States[' . $index . '][service_executable]', null, $state['project_service_executable'], [], false);
+	} ?></td>
+	<?php if (!empty($skip)): ?>
+		<td class="right"><?php if ($state['skippable_on_dependent']) {
```

Comment:
Regarding the missing validation: if the state is not marked as skippable, you won't be able to see whether flag is set on the state or not. So yeah, via UI you won't be able to create a configuration in which a state not marked as skippable is skipped. But if someone or something messes up your database, you wouldn't be able to tell, why states are skipped!?

So maybe show the checkbox if either skippable OR skip is set. And add a validation? Or show a disabled checkbox?